common/app/navigation/Navigation.scala

```diff
@@ -104,7 +104,10 @@ object NavMenu {
    * UsSportsPillar
    */
   private[navigation] def getChildrenFromOtherEditions(edition: Edition): Seq[NavLink] = {
-    Edition.others(edition).flatMap(edition => NavMenu.navRoot(edition).children ++ NavMenu.navRoot(edition).otherLinks)
+    // This shouldn't be a problem as Europe won't have special NavLinks
+    Edition
+      .othersWithBetaEditions(edition)
+      .flatMap(edition => NavMenu.navRoot(edition).children ++ NavMenu.navRoot(edition).otherLinks)
```

Comment:
I don't love this as we've introduced the idea of 'beta editions', but in using it here we are introducing a contract that beta editions shouldn't have special NavLinks.

If we can't get around this, perhaps we should make that contract more explicit in `Edition.scala` or document it somewhere?

Reply:
Fair point @OllysCoding - my understanding was that beta editions is a temporary construct until Europe is officially launched and will be removed once the experiment is over. If you think we will want to keep the logic around beta editions after the launch then we can definitely reconsider this